src/main/java/demon/EmptyDescriptionException.java

```diff
@@ -0,0 +1,11 @@
+package demon;
+
+/**
+ * An exception class that will be thrown when description of a task is not provided from the input.
+ */
+public class EmptyDescriptionException extends Exception {
```

Comment:
Nice exception, but perhaps you can try making a DemonException and inherit all demon-related exceptions from it? 👍

Reply:
Good idea, done!